store/storetest/reaction_store.go

```diff
@@ -21,10 +21,10 @@ func TestReactionStore(t *testing.T, ss store.Store) {
 }
 
 func testReactionSave(t *testing.T, ss store.Store) {
-	post := store.Must(ss.Post().Save(&model.Post{
```

Comment:
I would expect a `require.Nil(t, err)` check here and all the other `store.Must` replacements in this file